README.md

```diff
@@ -45,6 +46,14 @@ Download APK from GitHub Releases:
 Kotatsu is localized in a number of different languages, if you would like to help improve these or add new languages,
 please head over to the Weblate <a href="https://hosted.weblate.org/engage/kotatsu/">project page</a>
 
+## 😇 Contributing
+
+Whether you have ideas, translations, design changes, code cleaning, or real heavy code changes, help is always welcome.The more is done the better it gets!
```

Comment:
Looks like this is stolen from the [NewPipe`s old readme](https://github.com/TeamNewPipe/NewPipe/blob/212a413e93d17fd78bed9d22def4a5bc3294ed00/README.md#contribution)